save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/projectcollection/CollectionView.kt

```diff
@@ -4,49 +4,87 @@ package com.saveourtool.save.frontend.components.views.projectcollection
 
 import com.saveourtool.save.entities.ProjectDto
 import com.saveourtool.save.filters.ProjectFilters
+import com.saveourtool.save.frontend.TabMenuBar
 import com.saveourtool.save.frontend.components.RequestStatusContext
 import com.saveourtool.save.frontend.components.requestStatusContext
 import com.saveourtool.save.frontend.components.tables.TableProps
 import com.saveourtool.save.frontend.components.tables.columns
 import com.saveourtool.save.frontend.components.tables.tableComponent
 import com.saveourtool.save.frontend.components.tables.value
 import com.saveourtool.save.frontend.components.views.AbstractView
+import com.saveourtool.save.frontend.components.views.contests.tab
 import com.saveourtool.save.frontend.utils.*
 import com.saveourtool.save.frontend.utils.classLoadingHandler
 import com.saveourtool.save.info.UserInfo
+import com.saveourtool.save.validation.FrontendRoutes
 
 import csstype.ClassName
 import react.*
-import react.dom.html.ReactHTML.a
 import react.dom.html.ReactHTML.div
 import react.dom.html.ReactHTML.td
+import react.router.dom.Link
 
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 
+/**
+ * Enum that contains values for project
+ */
+@Suppress("WRONG_DECLARATIONS_ORDER")
+enum class ProjectsTab {
```

Comment:
ProjectListTab is even better (sorry came up with this name only now)

Reply:
Renames